www/_blog/2021-12-03-supabase-holiday-hackdays-hackathon.mdx

```diff
@@ -0,0 +1,135 @@
+
+Starting today at 7am PT we'll be hacking together for the next 10 days until 23:59pm PT on Sunday December 12th. And because we've launched [row-level security for our Realtime API](https://supabase.com/blog/2021/12/01/realtime-row-level-security-in-postgresql) earlier this week, we'd love for you to use it in your Hackathon project. May it be to build a [Santa Tracker](https://www.freecodecamp.org/news/create-your-own-santa-tracker-with-gatsby-and-react-leaflet/) or that secure family chat app to bring all your relatives together for the festive season, be creative and most importantly have fun 🥳
+
+[As always](https://supabase.com/blog/2021/10/14/hacktoberfest-hackathon-winners-2021#the-prizes), we'll have some extremely limited edition swag to give away that can only be won by participating in the Hackathon!
+
+The Supabase team will be participating as well, for example [Jon](https://twitter.com/_dijonmusters) and I, inspired by [Egghead's "Holiday Course Release Extravaganza"](https://egghead.io/20-days), will be building an Advent Calendar for all the [awesome community content](https://supabase.com/blog/2021/11/29/community-day#new-tutorials-and-integration-guides) you've been creating. Get ready for a cool open-source project which will be using a lot of the hottest Supabase features! To make sure you don't miss this one, follow us on [Twitter](https://twitter.com/supabase) and subscribe to our [YouTube channel](https://www.youtube.com/supabase). We'll see you there.
+
+## Details
+
+### Timeline
+
+- **Friday Dec 3rd at 07:00am PT:** Last day of Launch Week. Join us on [Twitter Spaces](https://twitter.com/i/spaces/1eaKbNDPEOYKX?s=20) for the kickoff announcement.
+- Build your project during the next 10 days and hang out with the community [on Discord](https://discord.gg/bnncdqgBSS).
+- **Sunday Dec 12th at 11:59pm PT:** Submission deadline
+- Judges Deliberate (Monday)
+- We'll be contacting and announcing the winners [on Twitter](https://twitter.com/supabase) throughout the week after.
+
+<blockquote class="twitter-tweet" data-theme="dark"><p lang="en" dir="ltr">Hacktoberfest Hackathon Swag finally arrived! 🥇🥈🥨 see the winners here: <a href="https://t.co/wgRBu92Bq4">https://t.co/wgRBu92Bq4</a> <a href="https://t.co/i1eD1pdA8I">pic.twitter.com/i1eD1pdA8I</a></p>&mdash; Supabase (@supabase) <a href="https://twitter.com/supabase/status/1463355998928736262?ref_src=twsrc%5Etfw">November 24, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+### Prize categories
+
+There are multiple chances to win, there will be prizes for:
+
+- Most impressive usage of the Realtime API features
+- Most fun / holiday themed project 
+
+There will be a winner and a runner-up prize for each category.
```

Comment:
add an image of the $1M gold t shirt?

Reply:
we have the supabase hacktoberfest swag tweet instead